### ISSUE TITLE
Rex-ray driver failed to initialize for Cinder

### ISSUE THREAD

**Issue by jranjan** [codedellemc/rexray#624]
I tried hard to follow the debugging tips available at google, but failed to solve the issue.

// Installed version
curl -sSL https://dl.bintray.com/emccode/rexray/install | sh -s -- stable 0.3.3

// rexray configuration
rexray:
  storageDrivers:
  - openstack
  volume:
    mount:
      preempt: true
openstack:
  authUrl: http://10.1.200.25:35357/v2.0/
  userID: 7cb8a726d58643f5827d3aa8b78ebebf
  userName: admin	
  password: secretword
  tenantName: admin
  regionName: RegionOne
  availabilityZoneName: nova

// Error

DEBU[0000] flattening env vars                           key=rexray.storageDrivers value=[openstack]
DEBU[0000] flattening env vars                           key=rexray.volumeDrivers value=[docker]
DEBU[0000] error initializing driver                     driverName=Openstack error.tenantName=admin error.domainName= error.userName=admin error.tenantId= error.domainId= error.provider=Openstack error.availabilityZone=nova error.userId=7cb8a726d58643f5827d3aa8b78ebebf error.password="******" error.region=RegionOne error.identityEndpoint="http://10.1.200.25:5000/v2.0/" error.instanceId=42244f88-52e5-860a-8824-b6bd32f6b6a8 error.moduleName=default-docker
INFO[0000] os driver initialized                         moduleName=default-docker provider=linux
INFO[0000] docker volume driver initialized              availabilityZone= iops= moduleName=default-docker provider=docker size= volumeRootPath=/data volumeType=
DEBU[0000] checking volume path cache setting            pathCache=true
INFO[0000] vdm.List                                      driverName=docker moduleName=default-docker
INFO[0000] listing volumes                               driverName=docker moduleName=default-docker
ERRO[0000] default module(s) failed to start
INFO[0000] service sent registered modules start signals
ERRO[0000] service initialized failed
INFO[0000] error initializing driver                     m=&{r:0xc8203557a0 name:default-docker addr:unix:///run/docker/plugins/rexray.sock desc:The default docker module.} m.r=&{Config:0xc8203652c0 OS:0xc820513ee0 Volume:0xc820500b60 Storage:0xc820513ef0 Context:default-docker drivers:map[VMAX:0xc82031b630 XtremIO:0xc820355800 Rackspace:0xc82035ebc0 ScaleIO:0xc820357b30 docker:0xc8200302f8 ec2:0xc820357da0 gce:0xc82035ea00 Isilon:0xc820368cd0 Openstack:0xc820355860 virtualbox:0xc820365600 linux:0xc820030300]}
root@ubuntu:~#

It will be great help if someone guide me.

**Comment by codenrhoden** [comment]
@koensayr It's worth nothing that we have still not re-introduced the OpenStack driver, even in 0.8.1. We have a lot of outstanding OpenStack issues, and we're waiting for the new driver to come in before seeing if those issues are still a problem.
